components/GlobalStlyle.js

```diff
@@ -0,0 +1,13 @@
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+*,*::after, *::before {
+    box-sizing:border-box;
+}
+body {
+    margin: 0;
+    padding: 0;
+}
+`;
+
+export default GlobalStyle;
```

Comment:
You already export the constant GlobalStyle in line 3. The default export in line 13 is redundant. You can remove one of them.